ui/app/styles/components/namespace-picker.scss

```diff
@@ -95,6 +60,11 @@
 
 .namespace-manage-link {
```

Comment:
No chance this could easily be solved with a helper instead of overriding the button for this specific case? We override elements and main classes a lot in components, and I want to push for helpers when practical.

Reply:
(Also, I wrote that CSS so blame me 100%)